libtiledbsoma/src/CMakeLists.txt

```diff
@@ -173,6 +173,7 @@ install(DIRECTORY
   ${CMAKE_CURRENT_SOURCE_DIR}/tiledbsoma
   ${CMAKE_CURRENT_SOURCE_DIR}/soma
   ${CMAKE_CURRENT_SOURCE_DIR}/utils
+  ${CMAKE_CURRENT_SOURCE_DIR}/../external/include/span
```

Comment:
Maybe external/ should be inside src/?  Or maybe CMakeLiists.txt should be one dir up?   Having a variable 'source dir' and then appending '..' feels a bit weird.